utils/helpers.js

```diff
@@ -60,10 +60,10 @@ const checkOculusBrowser = () => {
   // As of 5/26/22, only Oculus Browser has implemented the WebXR Hand Input Module and WebXR Layers API.
```

Comment:
Would you be able to write a quick doc / comments that explain the logic behind `Full`/`Partial`/`None` for the various checks?

Reply:
Yeah always helpful to have something there explicitly in comments